### DISCUSSION TITLE
Does Decky Loader officially support other distros?

### DISCUSSION THREAD

**Discussion by fsworld009** [SteamDeckHomebrew/decky-loader#687] [General]
Hi, just want to check if decky-loader is currently only officially supported for Steam Deck / SteamOS?

I'm asking because I observed issues only presented in Bazzite, not in SteamOS, but I'm not sure if it's appropriate to open an issue for this.

Would like to get a confirmation before I post about it, thanks!

**Comment by MurderFromMars** [comment]
it officially only supports steam os. 

but it is mostly functional on pretty much any distro. YMMV
